celesta-sql/src/main/java/ru/curs/celesta/score/discovery/ScoreByScorePathDiscovery.java

```diff
@@ -53,26 +54,21 @@ public ScoreByScorePathDiscovery(String scorePath) {
             Path scorePath = scoreDir.toPath().toAbsolutePath();
             return Files.walk(scorePath, FileVisitOption.FOLLOW_LINKS)
                     .filter(path -> Files.isRegularFile(path) && path.toString().endsWith(".sql"))
-                    .map(p -> {
-                        return new FileResource(p.toFile(), getNamespaceFromPath(scorePath.relativize(p)));
-                     })
+                    .map(p -> new FileResource(p.toFile(), getNamespaceFromPath(scorePath.relativize(p))))
                     .collect(Collectors.toCollection(LinkedHashSet::new));
         } catch (IOException ex) {
             throw new CelestaException(ex);
         }
     }
 
     Namespace getNamespaceFromPath(Path p) {
-        if (p.getNameCount() <= 1) {
-            return null;
-        }
-
-        StringBuilder sb = new StringBuilder(p.getName(0).toString());
-        for (int i = 1; i < p.getNameCount() - 1; i++) {
-            sb.append('.').append(p.getName(i).toString());
-        }
-
-        return new Namespace(sb.toString().toLowerCase());
+        return IntStream.range(0, p.getNameCount() - 1)
```

Comment:
I see you removed the `if (p.getNameCount() <= 1)`. Is there a unit case for this case?

Reply:
added